pwa/components/common/button/button.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { InputBase } from '../input';
+
+type buttonProps = {
+  text: string;
+  wide?: boolean;
+  empty?: boolean;
+  className?: string;
+  isRed?: boolean;
+};
+
+export const Button: React.FC<buttonProps> = ({ text, wide, empty, className, isRed }) => {
+  return (
+    <InputBase
+      type="submit"
+      value={text}
+      inputClassName={`
+      ${className}
+      font-semibold border-4
+      ${wide ? 'px-9 py-2' : 'px-3 py-0 h-fit'}
+      ${empty ? 'bg-eggshell' : isRed ? 'bg-red' : 'bg-argentinian_blue'}
+      ${isRed ? 'border-red' : 'border-argentinian_blue'}
+      ${!empty ? 'text-white' : isRed ? 'text-red' : 'text-argentinian_blue'}
+      ${empty && isRed ? 'text-red' : 'text-argentinian_blue'}`}
+    />
+  );
+};
```

Comment:
Why your button is returning an `InputBase` and not an HTML button?
- `empty` could be replaced by outline
- why `isRed` property?
- It's better to use variant

You could define a button component and override it with HTML properties (like the InputBase component)